src/components/LandingBody.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Stack, Button } from '@mui/material'; 
+import {Link} from 'react-router-dom'
+
+const LandingBody = () => {
+  return (
+    <section className="landing">
+    <div>
+      <div className="landing-inner">
+        <h1 className="x-large">Entrepreneur Connector</h1>
+        <p className='lead'>
+          A platform to connect with other entrepreneurs, VCs and peers.
+        </p>
+        <Stack direction='row' spacing={2}>
+          <Link to="/login" style={{textDecoration: 'none'}}><Button variant='contained' className="buttons">Login</Button></Link>
```

Comment:
try to create a separate class for this styling as it is used many times

Reply:
Acknowledged. Thanks.